pkg/unikontainers/unikontainers.go

```diff
@@ -509,13 +520,14 @@ func (u *Unikontainer) executeHook(hook specs.Hook, state []byte, wg *sync.WaitG
 }
 
 // ExecuteHooks executes sequentially any hooks found in spec based on name:
-func (u *Unikontainer) ExecuteHooksSequentially(name string) error {
+func (u *Unikontainer) executeHooksSequentially(name string) error { //nolint: unused
 	// NOTICE: This function is left on purpose to aid future debugging efforts
 	// in case concurrent hook execution causes unexpected errors.
 
 	// More info for individual hooks can be found here:
 	// https://github.com/opencontainers/runtime-spec/blob/main/config.md#posix-platform-hooks
-	Log.Info("Executing ", name, " hooks")
+	Log.WithField("hooks", name).Error("")
```

Comment:
Maybe this change can be reverted. There is no reason for error log level here.

Reply:
Yes, I changed the log level.